bulldozer/config_v1.go

```diff
@@ -53,7 +53,8 @@ type MergeConfig struct {
 }
 
 type MergeOption struct {
-	Body MessageStrategy `yaml:"body"`
+	Body                       MessageStrategy `yaml:"body"`
+	CommitDescriptionDelimiter *string         `yaml:"commit_description_delimiter,optional"`
```

Comment:
I'd call this `MessageDelimiter` (`message_delimiter` in YAML), make it a plain `string` (so it will be the empty string if unset), and remove the `optional` from the `yaml` tag.

Reply:
oh, I didn't know that primitives are allowed to be unset, good to know